docs/guides/update-guide/connectors/020-to-030.md

```diff
@@ -0,0 +1,117 @@
+---
+id: 020-to-030
+title: Update 0.2 to 0.3
+description: "Review which adjustments must be made to migrate from Connector SDK 0.2.x to 0.3.0."
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+<span class="badge badge--primary">Intermediate</span>
+
+The following sections explain which adjustments must be made to migrate from
+[Connector SDK](/components/connectors/custom-built-connectors/connector-sdk.md)
+0.2.x to 0.3.0.
+
+:::caution
+
+Be aware that the update from 0.2 to 0.3 requires manual migration steps as described below.
+
+:::
+
+## Connector function
+
+With SDK version 0.3.0, we introduce the following structural changes:
+
+- Input validation moves from Jakarta Bean Validation API version 3.0 to 2.0.
+- SDK artifacts have to be in scope `provided`.
+
+### Update to Validation API 2.0
+
+To better integrate in the current Java ecosystem and widely used frameworks like Spring 5 and Spring Boot 2, the `connector-validation` module
+now operates on Jakarta Bean Validation API version 2.0 instead of version 3.0. Adjust your Connector input objects using validation as follows:
+
+Replace all class imports starting with `jakarta.validation` by `javax.validation`. A Connector input class on SDK 0.2.x with the following imports:
+
+```java
+import io.camunda.connector.api.annotation.Secret;
+import jakarta.validation.constraints.NotEmpty;
```

Comment:
I would probably organize imports in _before_ and _after_ in the same order.

Reply:
Makes sense 👍